js/navigation.js

```diff
@@ -0,0 +1,77 @@
+namespacer('bcpl');
+
+bcpl.navigation = (($) => {
+	const navButtonSelector = '.nav-and-search nav button';
+	const navButtonAndListSelector = '.nav-and-search nav li.active button, .nav-and-search nav li.active ul';
+
+	const keyCodes = {
```

Comment:
Ignore this per this review, but it would be great to have a constants file in which we could pull this.